docker-compose-services/redis/README.md

```diff
@@ -60,27 +60,70 @@ $ ddev describe
 │          │      │ Host: localhost:54710                   │                       │
 ```
 
-## Examples
+This example shows:
 
-Frameworks, such as Laravel, require configuration to use the redis service.
+* From the webserver point-of-view:
+  * REDIS HOST: ddev-example-redis
+  * REDIS PORT: 6379
+
+* From the host Operating System point-of-view:
+  * REDIS HOST: localhost
+  * REDIS PORT: 54710
+
+## Framework Quickstarts
+
+* Frameworks require configuration to use the redis service.
+* You need to replace `<REDIS_HOST>` values with your project's redis host. Hint: `ddev describe`
+
+## Drupal
+
+* install the [`redis` module](https://www.drupal.org/project/redis)
+
+```shell
+ddev composer require drupal/redis
+```
+
+* enable the module via the website (`/admin/modules`) or via `drush` with the following command
+
+```shell
+$ ddev drush en redis
+ [success] Successfully enabled: redis
+```
+
+* add setting information to `web/sites/default/settings.ddev.php`; remember to update `<REDIS_HOST>`
+
+```php
+# DDEV SERVICE: Redis
+$settings['redis.connection']['interface'] = 'PhpRedis'; // Can be "Predis".
+$settings['redis.connection']['host']      = '<REDIS_HOST>';  // Your Redis instance hostname.
```

Comment:
You could use `redis` as the Redis service hostname as defined in here: https://github.com/drud/ddev-contrib/blob/master/docker-compose-services/redis/docker-compose.redis.yaml#L6 

```php
$settings['redis.connection']['host'] = 'redis';
$settings['redis.connection']['port'] = 6379;
```

same also on lines 114, and 118